defaults/defaults.go

```diff
@@ -116,3 +116,40 @@ var (
 	// CiliumPodSelector is the pod selector to be used for the Cilium agents.
 	CiliumPodSelector = "k8s-app=cilium"
 )
+
+// WellKnownImageDigests maps well known image paths and tags to their digests.
+// Do not remove old versions.
+var WellKnownImageDigests = map[string]string{
```

Comment:
What do you think on having this in a file and load it with `go:embed`?

Reply:
Nice idea, done.